tests/e2e/features/dicomViewer.feature

```diff
@@ -1,9 +1,16 @@
 Feature: Preview dicom image
+  As a user
+  I want to preview a DICOM file
+  so that I can preview DICOM image and its metadata
+
 
   Scenario: user previews dicom image
-    Given "Admin" has logged in
-    And "Admin" has uploaded the dicom file "MRBRAIN.dcm"
-    When "Admin" previews the dicom file "MRBRAIN.dcm"
-    And "Admin" opens files app
-    And "Admin" deletes file "MRBRAIN.dcm"
-    And "Admin" logs out
+    Given the user "Admin" has logged in
+    And the user has uploaded the dicom file "MRBRAIN.dcm"
+    When the user previews the dicom file "MRBRAIN.dcm"
+    And the user checks VIP metadata for dicom file
+    Then the user should see patient name "MR/BRAIN/GRASE/1024" in VIP metadata
+    When the user checks the extended metadata for dicom file
+    Then the user should see patient name "MR/BRAIN/GRASE/1024" in metadata sidebar
```

Comment:
Is this valid syntax for a scenario to have another `When` step after `Then`?

Reply:
Usually, we don't add another `When` after `Then`. But there might be some scenarios where we need to check for multiple things, in that case, we can use another `When` after `Then` step. I added multiple `When` and `Then` so that we can cover the tests in a single scenario.